osquery/tables/system/windows/services.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winsvc.h>
+#include <string>
+
+#include "osquery/tables/system/windows/registry.h"
```

Comment:
nit, place a space between local-search includes and global-search. Also, place local-search below global-search. This helps reduce potential bugs where global-search include files have implicit dependencies on things found locally. ;)